.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Please do NOT file bugs without filling in this form.
+title: '[Bug] '
+labels: ''
+assignees: ''
+
+---
+
+**Which version of Microsoft.IdentityModel are you using?**
+Note that to get help, you need to run the latest version. 
+<!-- E.g. Microsoft.IdentityModel 6.14 -->
+
+**Where is the issue?**
+ * [ ] M.IM.JsonWebTokens
```

Comment:
I'm not familiar with the type of issues opened in Wilson, so maybe you want something different listed here? in [id. web](https://github.com/AzureAD/microsoft-identity-web/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) we have them listed by scenario.